openmetadata-service/src/main/java/org/openmetadata/service/resources/analytics/ReportDataResource.java

```diff
@@ -25,20 +25,25 @@
 import lombok.extern.slf4j.Slf4j;
 import org.openmetadata.schema.analytics.ReportData;
 import org.openmetadata.schema.analytics.ReportData.ReportDataType;
+import org.openmetadata.schema.type.MetadataOperation;
+import org.openmetadata.service.Entity;
 import org.openmetadata.service.jdbi3.CollectionDAO;
 import org.openmetadata.service.jdbi3.ReportDataRepository;
 import org.openmetadata.service.resources.Collection;
 import org.openmetadata.service.security.Authorizer;
+import org.openmetadata.service.security.policyevaluator.OperationContext;
+import org.openmetadata.service.security.policyevaluator.ReportDataContext;
+import org.openmetadata.service.security.policyevaluator.ResourceContextInterface;
 import org.openmetadata.service.util.ResultList;
 
 @Slf4j
-@Path("/v1/analytic/reportData")
+@Path("/v1/analytics/reportData")
 @Api(value = "ReportData collection", tags = "ReportData collection")
```

Comment:
@TeddyCr we avoid using verbs in the method name. Can this be `/v1/analytics/report` or `v1/analytics/data`?